CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.14.0] - Unreleased
```

Comment:
For the final merge this will need to simply be unreleased and the version change in package.json to be reverted.